Tools/DumpRenderTree/TestOptions.cpp

```diff
@@ -67,6 +67,7 @@ const TestFeatures& TestOptions::defaults()
             // These are non-experimental WebPreference values that must always be set as they
             // differ from the default set in the WebPreferences*.yaml configuration.
             { "AllowsInlineMediaPlayback", true },
+            { "Canvas2DLayersEnabled", true },
```

Comment:
Why does this need to be set? New prefs really should not be using this.

Reply:
I didn't follow the new developments so what's the proper way to do this? I don't want to enable this pref by default, but still have it on during layout tests